package/MDAnalysis/lib/c_distances.pyx

```diff
@@ -315,3 +317,290 @@ def contact_matrix_pbc(coord, sparse_contacts, box, cutoff):
             if dist < cutoff2:
                 sparse_contacts[i, j] = True
                 sparse_contacts[j, i] = True
+
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def augment(float[:, ::1] coordinates, float[:,::1] box, float[:,::1] reciprocal, float r):
+    """Calculate augmented coordinate set
+    
+    Parameters
+    ----------
+    coordinates : array
+      Input coordinate array to generate duplicate images
+    box : array
+      Dimensions of the box of shape (3, 3)
+    reciprocal : array
```

Comment:
is this the reciprocal of `box`?  if so, we can just calculate it in the function